### ISSUE TITLE
"Generating an URL from non sanitized request parameters is insecure!"

### ISSUE THREAD

**Issue by RoxasShadow** [activeadmin/activeadmin#4292]
Rails 5.0.0.beta1.1
Ruby 2.3.0
ActiveAdmin rails-5-rspec

I tried to follow the [Oleander's suggestion](https://stackoverflow.com/questions/34413971/rails-5-0-0-beta1-generating-an-url-from-non-sanitized-request-parameters-is-i#comment57122824_34437768) to load the latest kaminari gem from GitHub itself but there are conflicts:

```
Resolving dependencies...
Bundler could not find compatible versions for gem "kaminari":
  In Gemfile:
    kaminari

    activeadmin was resolved to 1.0.0.pre2, which depends on
      kaminari (~> 0.15)

Could not find gem 'kaminari (~> 0.15)', which is required by gem 'activeadmin', in any of the sources.
```

**Comment by micred** [comment]
Use branch "0-17-stable".

Here there is my working Gemfile:


    gem 'devise', :github => 'plataformatec/devise', :ref => '1d77099861e2082a6bf5ef9c3a422c2ec315adb2'
   
    gem 'formtastic', github: 'justinfrench/formtastic', branch: 'master'  
    
    gem 'activeadmin', github: 'activeadmin/activeadmin', branch: 'master'
    
    gem 'ransack', github: 'activerecord-hackery/ransack', branch: 'master'
    
    gem 'kaminari', github: 'amatsuda/kaminari', branch: '0-17-stable'

**Comment by seanlinsley** [comment]
This should be fixed as of #4254. Be sure to follow the instructions in the [README](https://github.com/activeadmin/activeadmin#rails-5)
